appConfig.js

```diff
@@ -1,7 +1,7 @@
 const config = {
   appTitle: 'The New York Public Library',
   appName: 'Homepage',
-  favIconPath: '//d2znry4lg8s0tq.cloudfront.net/images/favicon.ico',
+  favIconPath: 'https://ux-static.nypl.org/images/favicon.ico',
```

Comment:
The relative url starting with `//` is better in case QA or other sites are not on https, then they won't get a mix-content issue. Everything else is okay.

Reply:
@EdwinGuzman, we should have everything serving over HTTPS. Use of the ambiguous protocol element has fallen out of practice since more and more search services and security suites are requiring everything be served over HTTPS.